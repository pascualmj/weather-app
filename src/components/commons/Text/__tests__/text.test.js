import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Text from "../Text";

describe("Component: Text", () => {
  test("Should match snapshot.", () => {
    const { asFragment } = render(<Text>hi</Text>);

    expect(asFragment()).toMatchSnapshot();
  });
});
