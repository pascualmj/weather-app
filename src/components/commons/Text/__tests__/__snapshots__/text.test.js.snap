// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Text Should match snapshot. 1`] = `
<DocumentFragment>
  <p
    class="text color-full size-regular weight-regular "
  >
    hi
  </p>
</DocumentFragment>
`;
